<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>step3</class>
 <widget class="QDialog" name="step3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bone Age</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>boneage_icon.png</normaloff>boneage_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(247, 242, 231);</string>
  </property>
  <widget class="QPushButton" name="label_change_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 107, 107);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(214, 214, 214);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;
border-top-left-radius: 8px;

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(194, 194, 194);
}</string>
   </property>
   <property name="text">
    <string>change_</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 173, 181);
border:none;
color: rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>YOLO x-ray roi label image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="result_frame">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>371</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(250, 250, 250);

color: rgb(214, 214, 214);
border-bottom-right-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 15px;
border-top-left-radius: 15px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="image2result">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(121, 122, 126);
</string>
   </property>
   <property name="text">
    <string>&gt;&gt;&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TRoBA">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 말랑말랑 Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(121, 122, 126);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(216, 211, 205);

border-bottom-right-radius: 5px;
border-bottom-left-radius: 5px;
border-top-right-radius: 5px;
border-top-left-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Test Result of Bone Age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_name2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(124, 124, 124);
padding-bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_Height2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>280</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(124, 124, 124);
padding-bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_gender2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(124, 124, 124);
padding-bottom: 7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Gender(F/M)</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_name">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txt_age">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txt_gender">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 242, 231);
border-color: rgb(121, 122, 126);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <widget class="QPushButton" name="print_button">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>690</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 말랑말랑 Regular</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 211, 205);

border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(50, 50, 50);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;
border-top-left-radius: 8px;

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(194, 194, 194);
}</string>
    </property>
    <property name="text">
     <string>print &gt;&gt;</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/newPrefix/dashboard2.ui</selectedon>
     </iconset>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="before_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>690</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 말랑말랑 Regular</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 211, 205);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(50, 50, 50);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;
border-top-left-radius: 8px;

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(194, 194, 194);
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt; BEFORE </string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(./img3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="time3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>한컴 말랑말랑 Regular</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 211, 205);</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1753</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd  hh:mm</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="input_img">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>411</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);
</string>
   </property>
   <property name="lineWidth">
    <number>-2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#797a7e;&quot;&gt;여기에 이미지가 출력됩니다.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="img_icon">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>140</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(./img5.png);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pred">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);
border:none;
border-bottom: 2px solid rgb(35, 35, 35);
color: rgb(124, 124, 124);
padding-bottom: 7px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>pred</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_gender_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>Bone</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_change_button</zorder>
  <zorder>description_label</zorder>
  <zorder>result_frame</zorder>
  <zorder>image2result</zorder>
  <zorder>TRoBA</zorder>
  <zorder>input_name2</zorder>
  <zorder>input_Height2</zorder>
  <zorder>input_gender2</zorder>
  <zorder>txt_name</zorder>
  <zorder>txt_age</zorder>
  <zorder>txt_gender</zorder>
  <zorder>input_img</zorder>
  <zorder>img_icon</zorder>
  <zorder>pred</zorder>
  <zorder>txt_gender_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
